### PR TITLE
Copy Edit

### PR DESCRIPTION
**Pull request by LindsayB610**
Assuming this works it'll be my edits to the post!

### CODE REVIEW

File: TUTORIAL.md [hunk 1]
```diff
@@ -1,51 +1,45 @@
...
 > Learn how to develop a microservice architecture with JHipster, Spring Boot, Angular, and MongoDB and deploy it all to the cloud using Kubernetes and Docker.
-
-This tutorial shows you how to build a microservices architecture with [JHipster 4.3.0](https://jhipster.github.io/2017/04/13/jhipster-release-4.3.0.html). 
-You'll generate a gateway (powered by Netflix Zuul and the JHipster Gateway), a microservice (that talks to MongoDB), and use Docker Compose to make
-sure it all runs locally. Then you'll deploy it to Minikube and Google Cloud using Kubernetes.
-
-{:toc}
-
-## What is JHipster?
+// Why is this set up like a quote, and why are you opening with it?
```
Comment (mraible) [inline TUTORIAL.md]: This was copied over from the README, can delete.

File: TUTORIAL.md [hunk 2]
```diff
@@ -1,51 +1,45 @@
...
-[the project on GitHub](https://github.com/jhipster/generator-jhipster), you can see it's mostly written in JavaScript 
-(39%) and Java (27%).
-
 At its core, JHipster is a [Yeoman](http://yeoman.io/) generator. Yeoman is a code generator that you run with a `yo` 
 command to generate complete applications or useful pieces of an application. Yeoman generators promote what the Yeoman 
 team calls the "`Yeoman workflow`". This is an opinionated client-side stack of tools that can help developers quickly 
 build beautiful web applications. It takes care of providing everything needed to get working without the normal pains 
 associated with a manual setup.
 
+*Learn more about JHipster, including it's origin, at [http://jhipster.github.io](http://jhipster.github.io).*
```
Comment (mraible) [inline TUTORIAL.md]: Typo: `it's` should be `its`

File: TUTORIAL.md [hunk 3]
```diff
@@ -1,51 +1,45 @@
...
+
+This tutorial shows you how to build a microservices architecture with [JHipster 4.3.0](https://jhipster.github.io/2017/04/13/jhipster-release-4.3.0.html). You'll generate a gateway (powered by Netflix Zuul and the JHipster Gateway), a microservice (that talks to MongoDB), and use Docker Compose to make sure it all runs locally. Then you'll deploy it to Minikube and Google Cloud using Kubernetes.
+
+// Reworked the intro, lmk what you think!
```
Comment (mraible) [inline TUTORIAL.md]: Looks good!

File: TUTORIAL.md [hunk 4]
```diff
@@ -1,51 +1,45 @@
...
+
+{:toc}
+// what does toc indicate here?
```
Comment (mraible) [inline TUTORIAL.md]: This macro works when using Asciidoctor. I was testing to see if it works with Markdown. It doesn't. ;)

File: TUTORIAL.md [hunk 5]
```diff
@@ -394,6 +375,7 @@ In this section, you'll change the following:
 2. Change the list of entries to look like a blog
 
 #### Allow HTML
+// Can you confirm somehow that our blog will support h4 styling?
```
Comment (mraible) [inline TUTORIAL.md]: Yep. It looks like an h3.

File: TUTORIAL.md [hunk 6]
```diff
@@ -786,6 +765,8 @@ but you don't need that for a Minikube deployment.
 
 8. Run `minikube service blog` to view the blog application. You should be able to login and add blogs, entries, and products.
 
+// I'm seeing a lot of list styles in this post that don't read like you, and aren't punctuated the way you typically punctuate lists. Is any of this post copy-pasted from elsewhere?
```
Comment (mraible) [inline TUTORIAL.md]: Yes: https://github.com/mraible/jhipster4-demo/blob/master/README.adoc and http://www.jhipster-book.com/#!/news/entry/build-jhipster-microservices-and-deploy-to-google-cloud-with-kubernetes

File: TUTORIAL.md (hunk 5 above)
Comment (LindsayB610) [inline TUTORIAL.md]: Is that a good thing?

File: TUTORIAL.md (hunk 6 above)
Comment (LindsayB610) [inline TUTORIAL.md]: they're all your copy tho, originally?

Comment (mraible) [inline TUTORIAL.md]: Yes.

File: TUTORIAL.md (hunk 2 above)
Comment (LindsayB610) [inline TUTORIAL.md]: whoops!

File: TUTORIAL.md (hunk 1 above)
Comment (LindsayB610) [inline TUTORIAL.md]: gotcha!

File: TUTORIAL.md (hunk 5 above)
Comment (mraible) [inline TUTORIAL.md]: Not sure, probably a conversation for the blog redesign.

**Pull request merged**